dashboard/test/ui/features/star_labs/maze2.feature

```diff
@@ -41,11 +41,11 @@ Feature: Complete a simple maze level
     Then element "#runButton" is visible
     And element "#resetButton" is hidden
     # move forward, Repeat: move forward, turn left, move forward
-    Then I drag block "1" to block "5"
-    And I drag block "4" to block "6"
-    And I drag block "1" into first position in repeat block "7"
-    And I drag block "2" to block "8"
-    And I drag block "1" to block "9"
+    Then I drag block "moveForward" to block "topBlock"
+    And I drag block "repeatForever" to block "moveForward"
+    And I drag block "turnLeft" into first position in repeat block "repeatForever"
+    And I drag block "moveForward" into first position in repeat block "repeatForever"
+    And I drag block "moveForward" to block "turnLeft"
```

Comment:
You'll notice that we need several `move forward` blocks for this program:
<img width="175" alt="image" src="https://github.com/code-dot-org/code-dot-org/assets/43474485/133f4cdc-aa22-4dd9-99ff-242160aba352">

However, we can only reliably predict the id for the first instance, placed by line 44 above. For this reason, I opted to place the `turn left` block inside the loop, and then add the second `move forward` block _above it_.

This results in the correct program but subtly changes the order in which the blocks are brought out. (Put another way, if the second `move forward` block were brought out first, we wouldn't be able to select it in order to place the `turn left` block below it.)

For a test like this where we're really interested in seeing the correct **feedback**, I'd suggest it could be more straight forward to simply provide a full program serialization and load it using Blockly APIs. Out of scope for now.